bibliopixel/__main__.py

```diff
@@ -0,0 +1,43 @@
+import platform
```

Comment:
What's the advantage of having a `__main__.py` over what we currently have?

Reply:
I've typically used `__main__.py` as the entry point for Python packages in development.
Typically you can run a directory using the following:
`python3 -m bibliopixel`
If you use all relative paths in your package (instead off `from bibliopixel import blah`, do `from . import blah`), you can even use `python3 bibliopixel`

It helps keep your entrypoints contained within the package itself. I prefer this route to help keep my code bundled together, and tend to keep the scripts directory for other purposes beside running the package.

You can also zip the package, and Python will run the `__main__.py` contained within the zip archive.
`python3 -m zipapp bibliopixel`
The zipapp method does require relative imports to be used as opposed to absolute, like if you would do `from bibliopixel import __version__`, it should really be `from . import __version__`

Reading up on `zipapp`, it does seem that zipapp writes its own `__main__.py` file that calls whatever function you set up as the entrypoint, however in other peps it does state that you should create a `__main__.py`. Not going to read more into it currently.

Some reading for anyone interested: 
https://www.python.org/dev/peps/pep-0338/
https://docs.python.org/3/library/zipapp.html
https://www.python.org/dev/peps/pep-0441/